model/Export/Stylesheet/AssetStylesheetLoader.php

```diff
@@ -26,6 +26,7 @@
 use League\Flysystem\FilesystemInterface;
 use oat\generis\model\OntologyAwareTrait;
 use oat\oatbox\filesystem\FileSystemService;
+use oat\taoMediaManager\model\fileManagement\FileManagement;
```

Comment:
I think this removal was in purpose to "enforce" the usage of `FlySystemManagement`. Check the return type of this method `FlySystemManagement` https://github.com/oat-sa/extension-tao-itemqti/pull/2211/files#diff-2e1fa53dc0a5951160dd0268a6376f80f4b93269b73dfc7dcb2a7e13a37f1defR98

Maybe we should instead use `FlySystemManagement::SERVICE_ID` there or change the return type? Cause the previous change seems to have broken the open/closed principle.

Reply:
Thx Gabriel. I should have taken more time to look at the replacement class signature.